ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ExternalBackupIntegrations/S3CompatibleIntegrationForm.tsx

```diff
@@ -28,9 +27,6 @@ import type { IntegrationFormProps } from '../../integrationFormTypes';
 
 import IntegrationFormActions from '../../IntegrationFormActions';
 import FormLabelGroup from '../../FormLabelGroup';
-import ScheduleIntervalOptions from '../../FormSchedule/ScheduleIntervalOptions';
-import ScheduleWeeklyOptions from '../../FormSchedule/ScheduleWeeklyOptions';
-import ScheduleDailyOptions from '../../FormSchedule/ScheduleDailyOptions';
```

Comment:
Looks like there are other integrations where we use the FormSelect. Might have to add a few more fixes to resolve those. Discovered the issue when I saw that the ScheduleIntervalOptions, ScheduleWeeklyOptions, and ScheduleDailyOptions components were still being used elsewhere. I'll have a follow up PR to convert the files to util files (like [this](https://github.com/stackrox/stackrox/pull/17732/files#diff-99c27b8a466fd10854abd550f396cba294f06a67daf0d4163c36ca7c758a8b0c)) and replace the FormSelect components to SingleSelect